### DISCUSSION TITLE
DTUs gehen nach einer Weile einer nach der anderen offline

### DISCUSSION THREAD

**Discussion by bynight66** [tbnobody/OpenDTU#1949] [Q&A]
Habe mir die Woche bei Diamex 4 openDTUs bestellt, weil ich eine Regelung für meine HM-800 Wechselrichter haben wollte. Eine DTU Pro arbeitet seit einem Jahr tadellos, nur lassen sich damit die Hoymiles nicht fernsteuern... 
Schönes großes Display... eingerichtet und funktioniert... aber nach ca. einer Stunde oder eher waren die ersten offline... 
ok... nach 2 Tagen Probiererei und SW aktualisieren war klar, es gab Probleme mit der Kommunikation mit der DTU Pro... Also heute, am Tag 3, DTU Pro ausgeschaltet und die 4 DTUs auf dem Boden ca. 5m unter den Wechselrichtern positioniert. Alle 4 DTUs an und auch alle ca. 20 Wechselrichter waren online. Ich dachte nun klappt's endlich. 4 IP Adressen und überall alle Daten am PC da... schön...
Nach ca. einer Stunde 3 Geräte offline und die Wechselrichter auf der Webseite rot. Nach ca. 2 Stunden alle 4 DTUs offline und alle Wechseltrichter rot. Bei meiner DTU Pro nie passiert... Das geht nun so seit 3 Tagen... So nützen mir die Dinger gar nichts, Die müssen ohne Ausfall von früh bis abends laufen! Laufen die bei Euch allen problemlos von früh bis abends?
Gruß, Ronald

**Comment by ghost** [comment]


**Reply by bynight66** [reply]
Hi Schneeer,

der Tip war gut! Haben alle dieselbe Seriennummer. Weiß nach dem updaten nicht mehr ob es daher rührt weil ich immer dieselbe config eingespielt habe oder ob es daher rührt weil der Hersteller immer dasselbe Image aufgespielt hat. 
Ich denke allerdings letzteres weil ich zu Beginn meine erstellte Copy Config mit der config einer neuen unkopierten DTU verglichen habe um eben zu vermeiden daß ich irgendwelche DTU spezifischen Daten wie eine einmalige Seriennummer überschreibe. Die Configs waren aber identisch, also keine unterschiedlichen Seriennummern.
Hab die Seriennummer jetzt überall um eines erhöht, also von 1 bis 5. Da bin ich ja gespannt ob das morgen was ändert... ;-)

Danke und Gruß,
Ronald

**Reply by bynight66** [reply]
Danke Scheeer!
Der Tip mit der Seriennummer war Gold wert! :-))) Aktuell alle 31 Wechselrichter an 5 openDTUs online! Ohne Neustart! Nicht mal ein einzelner offline, was ich die Tage zuvor auch oft hatte...
